components/Navbar.js

```diff
@@ -5,17 +5,92 @@ import Logo from '../public/logo.svg';
 import Button from '../components/form/Button';
 import Link from 'next/link';
 
+const style = {
+  container: `relative top-1/4 w-full text-left pl-16 mt-8`,
+  item: `text-3xl text-amcovad-white cursor-pointer  hover:amcovad-secondary-100`,
+  menu: {
+    isMenuOpen: `h-full w-full `,
+    close: `w-0 h-full`,
+    default: `overflow-x-hidden md:overflow-hidden transition-all duration-700 fixed z-10 top-0 left-0 bg-amcovad-primary-500`
+  }
+};
+
+const Menu = ({ children, isMenuOpen }) => {
+  return (
+    <div
+      className={`${style.menu.default} 
+       ${isMenuOpen ? style.menu.isMenuOpen : style.menu.close}`}
+    >
+      {children}
+    </div>
+  );
+};
+
+const MenuContainer = ({ children }) => {
+  return <div className={style.container}>{children}</div>;
+};
+
+const MenuItem = ({ href, children }) => {
+  return (
+    <div className="p-2">
+      <Link href={href} passHref>
+        <span className={style.item}>{children}</span>
+      </Link>
+    </div>
+  );
+};
+
 const Navbar = () => {
   const [isMenuOpen, setIsMenuOpen] = useState(false);
 
+  const toggle = () => {
```

Comment:
This function is not needed and is even more confusing as `toggle` doesn't sound right -- `toggleMenu` might be better but I don't think it is necessary.